web_source/cities/north_america/usa/denver.md

```diff
@@ -4,12 +4,14 @@ Country: USA (N. America)
 
 ##### Also known as:
 
-FIXME
+The Mile High City, Queen City of the Plains or Wall Street of the West.
```

Comment:
```suggestion
The kilometer High City, Queen City of the Plains or Wall Street of the West.
```

Reply:
I think we have to stick with the official name